x-pack/plugins/infra/server/lib/alerting/inventory_metric_threshold/evaluate_condition.ts

```diff
@@ -16,13 +16,26 @@ import { InfraSource } from '../../sources';
 import { calculateFromBasedOnMetric } from './lib/calculate_from_based_on_metric';
 import { getData } from './lib/get_data';
 
+interface AdditionalContextField {
+  [x: string]: any;
+}
+
+export interface AdditionalContext {
+  cloud?: AdditionalContextField;
```

Comment:
Does it mean we don't have information about the exact type or will it be improved later?

Reply:
We are only exposing top level variables - `host.*` or `cloud.*`. Anything inside host or cloud will become part of context. We don't know exactly what all fields would be indexed as I believe it depends on the beats app or any other source of data.